README.md

```diff
@@ -1,5 +1,48 @@
 # Open vSwitch CNI plugin
 
+This plugin allows user to define Kubernetes networks on top of Open vSwitch bridges available on nodes. IPAM is currently not supported. There is no scheduling involved, desired bridges must be precreated on all nodes. Also, ovs-cni does not configure bridges, it's up to user to connect them to L2, L3 or an overlay network. Finally please note that Open vSwitch must be installed and running on the host.
```

Comment:
at the end of the readme (appendix?) may be interesting to add example of manual configuration of host labels to allow for host selection by the pod

Reply:
I will create another documentation file with this under docs/.